include/deal.II/grid/manifold.h

```diff
@@ -269,6 +269,19 @@ class Manifold : public Subscriptor
                                        const Point<spacedim> &candidate) const;
 
   /**
+   * Given a point which lies close to the given manifold and to the given
+   * object, return one which lies on the Manifold.
+   *
+   * Internally this method calls the function project_to_manifold()
+   * passing the vertices of the object.
+   */
+  template<typename OBJ>
+  Point<spacedim>
+  project_to_manifold (const OBJ &obj,
```

Comment:
I see that that is the name also for the `get_default_quadrature` function. Patch for this forthcoming.